<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>objecttab</class>
 <widget class="QWidget" name="objecttab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QToolBox" name="objToolBox">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="objData">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>382</width>
        <height>220</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Page 1</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QListWidget" name="objDataList">
         <property name="styleSheet">
          <string notr="true">QListView::item:selected {
    background: #add8e6;
	color: #000;
}
QListView::item:selected:!active {
	background: #add8e6;
	color: #000;
}</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="objFunctions">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>382</width>
        <height>220</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Page 2</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTreeWidget" name="objFunctionsTree">
         <property name="styleSheet">
          <string notr="true">QTreeView::item:selected {
    background: #add8e6;
	color: #000;
}
QTreeView::item:selected:!active {
	background: #add8e6;
	color: #000;
}</string>
         </property>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
